projects/storefrontstyles/scss/components/cart/_add-to-cart.scss

```diff
@@ -13,7 +13,7 @@
       color: var(--cx-color-secondary);
     }
   }
-  .cx-btn {
+  .btn-relative {
```

Comment:
This is would be a breaking change meant for calydon theme, but it's not needed to fix the issue